static/app/components/organizations/pageFilterBar.tsx

```diff
@@ -1,10 +1,16 @@
 import styled from '@emotion/styled';
 
-const PageFilterBar = styled('div')`
+const PageFilterBar = styled('div')<{condensed?: boolean}>`
   display: flex;
   position: relative;
   border-radius: ${p => p.theme.borderRadius};
   height: ${p => p.theme.form.default.height}px;
+  ${p =>
+    p.condensed &&
+    `
+    max-width: 100%;
+    width: max-content;
+  `}
```

Comment:
nit
```suggestion
    css`
    max-width: 100%;
    width: max-content;
  `}
```

Reply:
ooh sorry this change was actually from a previous parent PR, I will make the change there though.